libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -218,7 +218,7 @@ void CodeTransform::operator()(VariableDeclaration const& _varDecl)
 			m_unusedStackSlots.erase(m_unusedStackSlots.begin());
 			m_context->variableStackHeights[&var] = slot;
 			if (int heightDiff = variableHeightDiff(var, varName, true))
-				m_assembly.appendInstruction(evmasm::swapInstruction(heightDiff - 1));
+				m_assembly.appendInstruction(evmasm::swapInstruction(unsigned(heightDiff) - 1));
```

Comment:
Is it certain that `heightDiff()` is never zero? Maybe you should assert here.

Reply:
`if (int heightDiff = variableHeightDiff(var, varName, true))` I think the if here makes it non zero inside if block?

Reply:
Oh, you're right. But that's one example why I dislike this style. `> 0` would make it so much clearer...